aux1.c

```diff
@@ -524,6 +535,11 @@ char *fromstring;
   kill_all_levels();
 #endif
   endgraf();
+
+  gen_save_fname(savestr);
```

Comment:
From change to `gen_save_filename` this call would be `gen_save_fname( savestr, 84 );`

Ignorable if we don't use `snprintf` in `gen_save_fname`